core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -17,7 +17,7 @@
 
 package kafka.coordinator.group
 
-import java.io.PrintStream
+import java.io.{IOException, PrintStream}
```

Comment:
Ah.. I just realized that I can not rely on intellij to show the unused import in gray color. It is fixed now.